components/AvalancheCenterForecastZonePolygons.tsx

```diff
@@ -13,7 +12,7 @@ export interface AvalancheCenterForecastZonePolygonsProps {
 }
 
 export const AvalancheCenterForecastZonePolygons: React.FunctionComponent<AvalancheCenterForecastZonePolygonsProps> = ({center_id, date, setRegion}) => {
-  const {isLoading, isError, data: mapLayer, error} = useMapLayer(center_id);
+  const {isLoading, isError, data: mapLayer /*, error */} = useMapLayer(center_id);
```

Comment:
Yikes - maybe let's log these at a minimum or send them to Sentry - this is one of those weird ones that can't really be shown since there's no place to put them ...

Reply:
Created https://github.com/stevekuznetsov/avalanche-forecast/issues/15 to look into this

Reply:
(we are currently logging validation errors to Sentry but network errors would also show up on this path, and we probably don't want those to go to Sentry)

Reply:
Sounds good!